client/rest-high-level/src/test/java/org/elasticsearch/client/eql/EqlSearchRequestTests.java

```diff
@@ -50,9 +52,26 @@ protected EqlSearchRequest createClientTestInstance() {
                 eqlSearchRequest.filter(QueryBuilders.termQuery(randomAlphaOfLength(10), randomInt(100)));
             }
         }
+        if (randomBoolean()) {
+            eqlSearchRequest.runtimeMappings(randomRuntimeMappings());
+        }
         return eqlSearchRequest;
     }
 
+    private Map<String, Object> randomRuntimeMappings() {
```

Comment:
Since we have this: https://github.com/elastic/elasticsearch/pull/71356/files#diff-b508afd1008661e9a36db7ed6e6e395aead3c2239c35a4d18c01b56c8c7aa471R306 in TestUtils, can't we use it?

Reply:
Unfortunately no.  Things in different projects (like this one and `sql-action`) are not accessible in a re-usable way.

Reply:
Actually, I take that back. I've added `ql:text-fixtures` as test dependency to rest-high-level project.

Reply:
Thx! that's nice!